src/api.js

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import {
+  favoriteUrl,
+  galleryVoatingUrl,
+  reportCommentAndReplyUrl,
+  createCommentUrl,
+  createReplyUrl,
+  voteForCommentAndReplyUrl,
+  albumDataUrl,
+  commentDataUrl,
+  userDetailsUrl,
+  userSettingUrl,
+} from './apiUrl';
+
+const accessToken = localStorage.getItem('access_token');
+
+const sendPost = (url, data = null) =>
+  axios.post(url, data, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+
+const sendGetPublic = url =>
+  axios(url, {
+    headers: {
+      Authorization: `Client-ID ${process.env.CLIENT_ID}`,
+    },
+  });
+
+const sendGetPrivate = url =>
+  axios(url, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+const sendPut = (url, data = null) =>
+  axios.put(url, data, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+
+// the get request who uses access token as header goes under sendGetPrivate
+const galleryVoatingApi = (galleryHash, vote) =>
+  sendGetPrivate(galleryVoatingUrl(galleryHash, vote));
+
+const favoriteAlbumApi = albumId => sendPost(favoriteUrl(albumId));
+
+const reportUserApi = (commentId, reason) =>
```

Comment:
we can remove `Api` in all function names. The function names should be a verb in general.

Reply:
https://github.com/Codebrahma/imgur-client-app/pull/15/commits/d98d2637c0efa7e5cfa622a37ef371cd20fcbb44
Done !